src/main/java/htsjdk/samtools/util/IntervalTree.java

```diff
@@ -114,6 +119,34 @@ public V put( final int start, final int end, final V value )
         return result;
     }
 
+    /**
+     * If the specified start and end positions are not already associated with a value or are
+     * associated with the sentinel ( see {@link #getSentinel()}, associates it with the given (non-sentinel) value.
+     * Otherwise, replaces the associated value with the results of the given
+     * remapping function, or removes if the result is equal to the sentinel value. This
+     * method may be of use when combining multiple values that have the same start and end position.
+     *
+     * @param start interval start position
+     * @param end interval end position
+     * @param value value to merge into the tree, must not be equal to the sentinel value
+     * @param remappingFunction a function that merges the new value with the existing value for the same start and end position
+     * @return the updated value that is now stored in the tree or sentinel
```

Comment:
`now` is not clear enough...(does it refer to before or after the application of the method?) 
perhaps `the updated value that is eventually stored in the tree or the sentinel value if nothing ends up being stored`

Reply:
I went with a slight modification:
```
the updated value that is stored in the tree after the completion of this merge operation, this will be the sentinel value if nothing ended up being stored
``